From 5751706d47e0f74c4f66e2eb24dfa24439c781ab Mon Sep 17 00:00:00 2001
From: Nic Costa <nic.costa@gmail.com>
Date: Sat, 9 Mar 2019 21:05:42 -0600
Subject: [PATCH] examples 0.8.0 should use mbed-edge 0.8.0

update the gitmodules ref to point to mbed-edge 0.8.0
instead of master
---
 .gitmodules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitmodules b/.gitmodules
index 09979c0..3080f4e 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -5,4 +5,4 @@
 [submodule "lib/mbed-edge"]
 	url = git@github.com:ARMmbed/mbed-edge.git
 	path = lib/mbed-edge
-	branch = refs/heads/master
+	branch = refs/tags/0.8.0
-- 
2.7.4

